pressbooks/pressbooks-book-directory-fe
# Collection BookCard buttons

This PR Solves #171 

### How to test

1. Select collections and toggle the collections from the buttons on the BookCard
2. Try to activate collections from the facet filters accordion and it should work as expected too
